src/kixi/data_acquisition/db/inmemory.clj

```diff
@@ -0,0 +1,40 @@
+(ns kixi.data-acquisition.db.inmemory
```

Comment:
these in memory stubs always worry me as we often aren't doing the same shape of data storage when we use c* or others.